test/snippet/core/algorithm/configuration_get.cpp

```diff
@@ -9,13 +8,13 @@ int main()
 {
     using seqan3::get;
 
-    seqan3::configuration my_cfg = seqan3::align_cfg::gap{seqan3::gap_scheme{seqan3::gap_score{-1},
-                                                                             seqan3::gap_open_score{-10}}} |
+    seqan3::configuration my_cfg = seqan3::align_cfg::gap_cost_affine{seqan3::align_cfg::open_score{-10},
+                                                                      seqan3::align_cfg::extension_score{-1}} |
                                    seqan3::align_cfg::band_fixed_size{seqan3::align_cfg::lower_diagonal{-4},
                                                                       seqan3::align_cfg::upper_diagonal{4}};
-    // my_cfg is now of type configuration<gap, band>
+    // my_cfg is now of type configuration<gap_cost_affine, band_fixed_size>
 
-    seqan3::debug_stream << get<1>(my_cfg).lower_diagonal                                  << '\n';  // prints -4
-    seqan3::debug_stream << get<seqan3::align_cfg::band_fixed_size>(my_cfg).upper_diagonal << '\n';  // prints 4
-    seqan3::debug_stream << get<seqan3::align_cfg::gap>(my_cfg).value.get_gap_score()      << '\n';  // prints -1
+    seqan3::debug_stream << get<1>(my_cfg).lower_diagonal                                         << '\n';  // prints -4
+    seqan3::debug_stream << get<seqan3::align_cfg::band_fixed_size>(my_cfg).upper_diagonal        << '\n';  // prints 4
```

Comment:
```suggestion
    seqan3::debug_stream << get<1>(my_cfg).lower_diagonal << '\n';  // prints -4
    seqan3::debug_stream << get<seqan3::align_cfg::band_fixed_size>(my_cfg).upper_diagonal << '\n';  // prints 4
```

T_T those whitespaces